.github/ISSUE_TEMPLATE/epic.yaml

```diff
@@ -0,0 +1,89 @@
+name: Epic (Internal Use Only 🔒)
+description: Define an initiative that spans multiple user stories.
+title: "[Epic]: <DESCRIPTIVE TITLE HERE>"
+labels: ["type/epic"]
+assignees:
+  - octocat
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ## ⚠️ Warning
+        Issues that do not follow the below template will be marked `status/back-to-author` and assigned back to the creator.
+        
+        ---
+        
+        # 📜 Epic Details
+  - type: textarea
+    id: context
+    attributes:
+      label: Background story
+      description: What's the big idea with this epic?
+      placeholder: Be specific! Ctrl+Copy screenshots!
+    validations:
+      required: true
+  - type: textarea
+    id: accomplishment
+    attributes:
+      label: What exactly will this accomplish?
+      description: Provide a detailed list of your desired outcomes.
+      placeholder: Be specific! Use bullet-points liberally.
+    validations:
+      required: true
+  - type: textarea
+    id: user-stories
+    attributes:
+      label: 'Child stories'
+      description: "What atomic stories will make this epic a reality? Titles only."
+      value: |
+        1.
+        2.
+        3.
+      render: bash
+    validations:
+      required: true
+  - type: input
+    id: which-services
+    attributes:
+      label: Which AWS service(s)?
+      description: List out the [AWS service(s)](https://aws.amazon.com/products/) this code will engage.
+      placeholder: Be specific!
+    validations:
+      required: true
+  - type: checkboxes
+    id: lingo-type
+    attributes:
+      label: Which AWS SDK's or tools?
+      options:
+        - label: All languages
+        - label: AWS CLI
+        - label: Java
+        - label: Kotlin
+        - label: Python
+        - label: C++
+        - label: Ruby
+        - label: .NET
+        - label: Javascript
+        - label: Java v2
+        - label: Javascript v3
+        - label: Go
+        - label: PHP
+        - label: Rust
+        - label: Swift
+        - label: Typescript
+        - label: Not applicable
+  - type: textarea
+    id: which-examples
+    attributes:
+      label: Is there any existing code that could be leveraged?
+      description: Check [here](https://docs.aws.amazon.com/code-samples/latest/catalog/welcome.html) for possible scaffolding or foundation.
```

Comment:
Link to the [Code Library](https://docs.aws.amazon.com/code-library/latest/ug/what-is-code-library.html) instead of the code catalog.

Reply:
Done.

Reply:
Not done? 🤔